OpenTap.Metrics/AssetDiscovery/IResourceDiscovery.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
```

Comment:
I would make sense to rename the files as well.

Reply:
Yes ofcause. Thanks.